vdom/core.py

```diff
@@ -14,6 +14,8 @@
 import os
 import io
 
+from vdom.frozendict import FrozenDict
```

Comment:
This doesn't exist -- did you write this in a file that isn't included in a commit here or is this supposed to come from a dependency?

Reply:
Yeah, I missed adding it. Done now